build.gradle

```diff
@@ -319,6 +319,7 @@ dependencies {
 
   implementation group: 'org.springdoc', name: 'springdoc-openapi-starter-webmvc-ui', version: '2.1.0'
   implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign', version: '4.0.3'
+  implementation 'org.springframework.retry:spring-retry:2.0.2'
```

Comment:
i thought this would mean that it would always use the latest version, and so a breaking change may be introduced and the developer may not know where ti came from?

Reply:
Ah it's not being pulled in through `spring-bom`, so please disregard this comment.